README.md

```diff
@@ -79,6 +85,27 @@ exports.handler = handler;
 Tasks that use this library are just standard AWS Lambda tasks. Information on
 creating release packages is available [here](https://docs.aws.amazon.com/lambda/latest/dg/deployment-package-v2.html).
 
+## Usage in Cumulus Deployments
+
+During deployment, Cumulus will automatically obtain and inject the [Cumulus Message Adapter](https://github.com/cumulus-nasa/cumulus-message-adapter) zip into the compiled code and create a zip file to be deployed to Lambda.
```

Comment:
I'd probably remove the word "zip", since that's an implementation detail that we could change in the future without task developers knowing or caring about it.

Reply:
Done.